tests/test_bulk_update.py

```diff
@@ -397,6 +397,31 @@ def test_auto_now(self):
         self.assertEqual(datetime(2020, 1, 2, 0, 0, 0,  tzinfo=tz_utc), instance.checked)
         self.assertEqual(instance.updated, get_auto_now_date())
 
+    def test_auto_now_respects_override(self):
```

Comment:
there should also be similar tests for `bulk_update_or_create` and `bulk_create`

Reply:
Done